.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Set up PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '7.4'
+          php-version: '8.0'
```

Comment:
We want to lint with 7.4 to make sure we don't use any 8+ syntax accidentally.

Reply:
ok